tox.ini

```
@@ -1,21 +1,17 @@
 [tox]
 isolated_build = True
 envlist =
-    py{38,39,310}-dj32
-    py{38,39,310}-dj40
-    py{38,39,310,311}-dj41
     py{38,39,310,311,312}-dj42
     py{310,311,312}-dj50
+    py{310,311,312}-dj51
     py{310,311,312}-djmain
 
 [testenv]
 pip_pre = True
 deps =
-    dj32: django~=3.2.0
-    dj40: django~=4.0.0
-    dj41: django~=4.1.0
     dj42: django~=4.2.0
     dj50: django~=5.0.0
+    dj51: django>=5.1,<5.2
```

```suggestion
    dj51: django~=5.1b
```

This one works on my local machine. Sorry for the false lead.